modules/appeals_api/app/controllers/appeals_api/schemas/shared_schemas_controller.rb

```diff
@@ -48,16 +38,8 @@ def schema_type
     @schema_type ||= params[:schema_type]
   end
 
-  def shared_schemas_file
```

Comment:
I noticed that we had some duplicative code around loading shared schemas in this file and in the `lib/form_schemas.rb` file, and a little bit in `rswag_config.rb`.

I decided to localize the logic in `form_schemas.rb`, and reused that localized code here and in `rswag_config.rb`.